pkg/openai/client.go

```diff
@@ -121,8 +162,9 @@ func (c *Client) ListModels(ctx context.Context, providers ...string) (result []
 		return nil, nil
 	}
 
+	// If auth is invalid, we just want to return nothing.
```

Comment:
nit: could you amend the comment to explain why we want to do nothing?

Reply:
Done